autospec/avxjudge.py

```diff
@@ -0,0 +1,512 @@
+    avx2_instructions.append("vfmsub231ss")
+    avx2_instructions.append("vfmsub132sd")
+    avx2_instructions.append("vfmsub231sd")
+    avx2_instructions.append("vfmsub213sd")
+
+    avx2_instructions.append("vfnmadd132ss")
+    avx2_instructions.append("vfnmadd213ss")
+    avx2_instructions.append("vfnmadd231ss")
+    avx2_instructions.append("vfnmadd132sd")
+    avx2_instructions.append("vfnmadd231sd")
+    avx2_instructions.append("vfnmadd213sd")
+
+    avx2_instructions.append("vfnmsub132ss")
+    avx2_instructions.append("vfnmsub213ss")
+    avx2_instructions.append("vfnmsub231ss")
+    avx2_instructions.append("vfnmsub132sd")
+    avx2_instructions.append("vfnmsub231sd")
+    avx2_instructions.append("vfnmsub213sd")
+
+    avx2_instructions_hv.append("vpclmulhqlqdq")
+    avx2_instructions_hv.append("vpclmullqhqdq")
+
+    avx2_instructions_hv.append("vfmadd132ps")
+    avx2_instructions_hv.append("vfmadd213ps")
+    avx2_instructions_hv.append("vfmadd231ps")
+    avx2_instructions_hv.append("vfmadd132pd")
+    avx2_instructions_hv.append("vfmadd231pd")
+    avx2_instructions_hv.append("vfmadd213pd")
+    avx2_instructions_hv.append("vfmsub132ps")
+    avx2_instructions_hv.append("vfmsub213ps")
+    avx2_instructions_hv.append("vfmsub231ps")
+    avx2_instructions_hv.append("vfmsub132pd")
+    avx2_instructions_hv.append("vfmsub231pd")
+    avx2_instructions_hv.append("vfmsub213pd")
+
+    avx2_instructions_hv.append("vfnmadd132ps")
+    avx2_instructions_hv.append("vfnmadd213ps")
+    avx2_instructions_hv.append("vfnmadd231ps")
+    avx2_instructions_hv.append("vfnmadd132pd")
+    avx2_instructions_hv.append("vfnmadd231pd")
+    avx2_instructions_hv.append("vfnmadd213pd")
+    avx2_instructions_hv.append("vfnmsub132ps")
+    avx2_instructions_hv.append("vfnmsub213ps")
+    avx2_instructions_hv.append("vfnmsub231ps")
+    avx2_instructions_hv.append("vfnmsub132pd")
+    avx2_instructions_hv.append("vfnmsub231pd")
+    avx2_instructions_hv.append("vfnmsub213pd")
+    avx2_instructions_hv.append("vdivpd")
+
+
+    return
+
+def is_sse(instruction:str, args:str) -> float:
+
+    val: float = -1.0
+    if "xmm" in args:
+        if ("pd" in instruction or "ps" in instruction or instruction in sse_instructions_xmm):
+            val = 1.0
+        else:
+            val = 0.01
+    return val
+
+
+def is_avx2(instruction:str, args:str) -> float:
+    val: float = -1.0
+
+    if "ymm" in args:
+        if ("pd" in instruction or "ps" in instruction or instruction in avx2_instructions_ymm) and "xor" not in instruction and "vmov" not in instruction:
+            val = 1.0
+        else:
+            val = 0.01
+
+    if instruction in avx2_instructions_lv:
+        val = max(val, 0.1)
+    if instruction in avx2_instructions:
+        val = max(val, 1.0)
+    if instruction in avx2_instructions_hv:
+        val = max(val, 2.0)
+
+    return val
+
+def has_high_register(args: str) -> int:
+    if "mm16" in args:
```

Comment:
Ok, this is not how python works. Here's a little lesson
```python
ls = [1, 2, 3, 4]
if any(l in ints for l in ls):
    return True
```
Also, use a bool! They exist natively in Python!